spec.emu

```diff
@@ -20,54 +20,123 @@ contributors: Ujjwal Sharma, Younies Mahmoud
       <table class="real-table">
         <thead>
           <tr>
-            <th>Internal Slot</th>
-            <th>Property</th>
+            <th>Value Slot</th>
+            <th>Style Slot</th>
+            <th>Display Slot</th>
+            <th>Unit</th>
           </tr>
         </thead>
         <tr>
           <td>[[Years]]</td>
+          <td>[[YearsStyle]]</td>
+          <td>[[YearsDisplay]]</td>
           <td>`"years"`</td>
         </tr>
         <tr>
           <td>[[Months]]</td>
+          <td>[[MonthsStyle]]</td>
+          <td>[[MonthsDisplay]]</td>
           <td>`"months"`</td>
         </tr>
         <tr>
           <td>[[Weeks]]</td>
+          <td>[[WeeksStyle]]</td>
+          <td>[[WeeksDisplay]]</td>
           <td>`"weeks"`</td>
         </tr>
         <tr>
           <td>[[Days]]</td>
+          <td>[[DaysStyle]]</td>
+          <td>[[DaysDisplay]]</td>
           <td>`"days"`</td>
         </tr>
         <tr>
           <td>[[Hours]]</td>
+          <td>[[HoursStyle]]</td>
+          <td>[[HoursDisplay]]</td>
           <td>`"hours"`</td>
         </tr>
         <tr>
           <td>[[Minutes]]</td>
+          <td>[[MinutesStyle]]</td>
+          <td>[[MinutesDisplay]]</td>
           <td>`"minutes"`</td>
         </tr>
         <tr>
           <td>[[Seconds]]</td>
+          <td>[[SecondsStyle]]</td>
+          <td>[[SecondsDisplay]]</td>
           <td>`"seconds"`</td>
         </tr>
         <tr>
           <td>[[Milliseconds]]</td>
+          <td>[[MillisecondsStyle]]</td>
+          <td>[[MillisecondsDisplay]]</td>
           <td>`"milliseconds"`</td>
         </tr>
         <tr>
           <td>[[Microseconds]]</td>
+          <td>[[MicrosecondsStyle]]</td>
+          <td>[[MicrosecondsDisplay]]</td>
           <td>`"microseconds"`</td>
         </tr>
         <tr>
           <td>[[Nanoseconds]]</td>
+          <td>[[NanosecondsStyle]]</td>
+          <td>[[NanosecondsDisplay]]</td>
           <td>`"nanoseconds"`</td>
         </tr>
       </table>
     </emu-table>
 
-    <emu-note type=editor>The [[Weeks]] field is still under discussion in Temporal. For more, check out the <a href="https://github.com/tc39/proposal-temporal/issues/532">Github thread</a> on the Temporal issue tracker.</emu-note>
+    <emu-clause id="sec-getunitoptions" aoid="GetUnitOptions">
+      <h1>GetUnitOptions ( _unit_, _options_, _baseStyle_, _prevStyle_ )</h1>
+      <p>The GetUnitOptions abstract operation is called with the arguments _unit_ (which must be a String), _options_ (which must be an Object), _baseStyle_ (which must be a String), and _prevStyle_ (which must be a String) and returns a Record containing the relevant options for that unit. The following steps are taken:</p>
```

Comment:
> `_prevStyle_` (which must be a String)

`prevStyle` can also be undefined, right?

Reply:
yes, but I'm not sure if we need to mention this here. *Anything* can be undefined, right?